orchestrator-salt/src/main/resources/salt/salt/postgresql/disaster_recovery/scripts/backup_db_compressed.sh

```diff
@@ -0,0 +1,254 @@
+#!/bin/bash
```

Comment:
:exclamation: to reviewers:
this file is mostly a copy-paste from existing `backup_db.sh` file. Differences will be highlighted. Unfortunately merging compress/non-compress functionality in one file would make it hard to read and error-prone. We should be able to use the original `backup_db.sh` file at any time without worry of potential change in functionality.